app/components/modal-error/error.modal.tsx

```diff
@@ -0,0 +1,111 @@
+import React, { Component } from 'react';
+import { Modal, View, StyleSheet, Dimensions, TouchableOpacity} from 'react-native';
+import VectorIcon from 'react-native-vector-icons/FontAwesome';
+import { color } from '../../theme';
+import { Text } from '../text/text';
+const { width: SCREEN_WIDTH } = Dimensions.get('window');
+const ALERT_WIDTH = SCREEN_WIDTH - (SCREEN_WIDTH / 4);
+
+interface IErrorModalProps {
+	visible: boolean,
+	message: string,
+	handelClose: Function
+}
+
+
+class ErrorModal extends Component<IErrorModalProps> {
+	constructor (props: IErrorModalProps) {
+		super(props);
+	}
+
+	render() {
+		return (
+			<Modal visible={this.props.visible} animationType="fade" transparent onRequestClose={() => null }>
+				<View style={styles.container}>
+					<View style={[styles.iconCircle, { top: 32, borderColor: color.palette.white }]}>
+						<View style={styles.iconCircleTop}>
+							<VectorIcon name="close" size={20} color='white' />
+						</View>
+					</View>
+
+					<View style={[styles.content, { backgroundColor: color.palette.white }]}>
+						<View style={[styles.iconCircle, { top: -32, borderColor: color.palette.white }]}>
+							<View style={styles.iconCircleTop}>
+								<VectorIcon name="close" size={20} color='white' />
+							</View>
+						</View>
+
+						<Text style={styles.message}>{this.props.message}</Text>
+						<TouchableOpacity style={styles.modalButton}
+										onPress={()=>this.props.handelClose()}>
+							<Text style={{color : color.palette.white, textAlign: 'center'}}>OK</Text>
+						</TouchableOpacity>
+					</View>
+				</View>
+			</Modal>
+		)
+	}
+}
+export default ErrorModal;
+
+const styles = StyleSheet.create({
+		container: {
+			position: 'absolute',
```

Comment:
btw, instead of those 4 lines you can use ...stylesheet.absoulteFill

Reply:
nice! i ignored this style , nevertheless, this component will go to the garbage this week, due to new UI